src/core/types/State.ts

```diff
@@ -241,3 +243,8 @@ export interface UserState {
     getNftBalance: Status;
   };
 }
+
+export interface PartnerState {
+  id: PartnerId | undefined;
+  address: Address | undefined;
```

Comment:
```suggestion
  id?: PartnerId;
  address?: Address;
```

Reply:
We set the state types intentionally not optional just so that we have to explicitly define the whole initial state on the reducer and all its properties, for readability and avoiding errors.

I would agree with changing to `null`, but would rather make that refactor for all state in this file in a separate PR, just so we have a convention, as it will be hard to know for other contributors which one to use.

Will leave it as is for now since it does not affect functionality.